src/apps/backoffice/backoffice.element.ts

```diff
@@ -1,5 +1,5 @@
 import { UmbBackofficeContext, UMB_BACKOFFICE_CONTEXT_TOKEN } from './backoffice.context.js';
-import { UmbExtensionInitializer } from './extension-initializer.controller.js';
+import { UmbServerExtensionRegistrator } from './server-extension-registrator.controller.js';
```

Comment:
Its because it's asking the server for extensions and registering them.  The registration is done on the registry, same for everyone. This one just makes the call to the server and parses the data.